src/Evolution/Executables/GeneralizedHarmonic/EvolveGeneralizedHarmonicFwd.hpp

```diff
@@ -5,4 +5,19 @@
 
 #include <cstddef>
 
+/// \cond
```

Comment:
So I can understand, why are these forward declarations needed now but not before this PR?

Reply:
Because before the `EvolutionMetavars` didn't have template parameters, but hard-coded the initial data and boundary conditions. Now they are chosen in cmake, so they need to be forward-declared.